config/foreman.hiera/tuning.yaml

```diff
@@ -16,3 +16,4 @@ postgresql::server::config_entries:
   max_connections: 500
   shared_buffers: 512MB
   work_mem: 4MB
+  log_line_prefix: '%t'
```

Comment:
I *think* there's a space missing, it was `'%t '` before, wasn't it?

Reply:
yeah, definitely need that space